packages/atomic/src/components/result-lists/list-display-results.tsx

```diff
@@ -12,16 +12,14 @@ export const ListDisplayResults: FunctionalComponent<ResultsProps> = (
         result={result}
         engine={props.bindings.engine}
         store={props.bindings.store}
-        display={props.display}
-        density={props.density}
-        imageSize={props.imageSize}
```

Comment:
Where does `atomic-result` receive the list's display/density/imageSize after this change?